routes/api.php

```diff
@@ -14,7 +14,7 @@
 */
 
 
-Route::group(['prefix' => 'v1','namespace' => 'Api', 'middleware' => 'auth:api'], function () {
+Route::group(['prefix' => 'v1','namespace' => 'Api', 'middleware' => ['api', 'throttle:'.config('app.api_throttle_per_minute').',1']], function () {
```

Comment:
If we can say this here, can we instead say this in the `RouteServiceProvider`? I feel that's "higher-up" and thus the more appropriate place to put this.

And it means you can delete the 'middleware' clause from here.

Reply:
I tried that and it didn't work. I can take another stab tho.

Reply:
I had eyes on it just the other day, let me see if I can give you a push.